lib/src/components/position_component.dart

```diff
@@ -58,24 +67,22 @@ abstract class PositionComponent extends BaseComponent {
   }
 
   /// Get the position that everything in this component is positioned in relation to
-  Vector2 get absoluteCanvasPosition {
+  Vector2 get absoluteParentPosition {
     if (parent is PositionComponent) {
       return (parent as PositionComponent).absoluteTopLeftPosition;
     } else {
       return Vector2.zero();
     }
   }
 
-  /// Set the top left position regardless of the anchor
-  set topLeftPosition(Vector2 position) {
-    this.position = position + (anchor.toVector2..multiply(size));
-  }
-
-  /// Get the position of the center of the component
+  /// Get the position of the center of the component without rotation
```

Comment:
if you didn't read the whole Hitbox docs, it's not clear which center is here. I would briefly define center as the center of the containing rectangle (If you have complex shapes there are many kinds of centers)